bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/Channel.java

```diff
@@ -49,24 +54,29 @@ public Channel(ChannelUID uid, String acceptedItemType) {
         this.uid = uid;
         this.acceptedItemType = acceptedItemType;
         this.configuration = new Configuration();
+        this.properties = Collections.unmodifiableMap(new HashMap<String, String>(0));
```

Comment:
The properties of the thing are mutable. So in order to be compliant I would expect the same for the Channel. This is also means, that we need a "setProperty" method similar to the Thing.